### ISSUE TITLE
Add unique_id to sensor.backup_state

### ISSUE THREAD

**Issue by holdestmade** [sabeechen/hassio-google-drive-backup#1005]
Please add `unique_id` to `sensor.backup_state` so an area can be assigned in the UI

**Comment by Krispkiwi** [comment]
seconded, it's one of the only things in my setup without a unique id on the sensors.

**Comment by MarcusSRussell** [comment]
Agreed - it’s a minor th8ng I appreciate but would just make what is a very tidy and useful add on

**Comment by jhenkens** [comment]
See https://github.com/sabeechen/hassio-google-drive-backup/issues/579 and my latest comment there.
